src/Topologies/dtopology2d.jl

```diff
@@ -84,6 +84,30 @@ struct DistributedTopology2D{
 
     "a NamedTuple of vectors of `(e,face)` "
     boundaries::BF
+    "internal local elements (lidx)"
+    internal_elems::Vector{Int}
+    "local elements (lidx) located on process boundary"
+    perimeter_elems::Vector{Int}
+    "total number of unique non-boundary vertices"
+    nglobalvertices::Int
+    "total number of unique non-boundary faces"
+    nglobalfaces::Int
+    "global id for each unique ghost vertex"
+    ghost_vertex_gidx::Vector{Int}
+    "global id for each unique ghost face"
+    ghost_face_gidx::Vector{Int}
+    "number of unique vertices to be communicated to neighboring processes"
+    comm_vertex_lengths::Vector{Int}
```

Comment:
`comm_vertex_lengths[i]` = number of unique vertices to be communicated to `neighbor_pids[i]`